Entities/Attachable.h

```diff
@@ -609,6 +609,34 @@ ENTITYALLOCATION(Attachable)
 
 	virtual void SetInheritsRotAngle(bool inherit) { m_InheritsRotAngle = inherit; }
 
+
+/// <summary>
+/// Whether this attachable is set to collide with terrain when attached to a parent or not.
+/// </summary>
+/// <return>Whether this attachable is set to collide with terrain or not.</return>
```

Comment:
See above stuff about m_IsSetToCollide, if we have to keep this stuff, this naming isn't really clear on the different between IsSetToCollide and CollidesWithTerrain.